lib/openstack_query/server_manager.py

```diff
@@ -0,0 +1,167 @@
+from typing import List, Dict, Any, Union
+from structs.query.query_details import QueryDetails
+from structs.query.preset_details import PresetDetails
+
+from enums.query.query_presets import QueryPresets
+from enums.query.server.server_properties import ServerProperties
+
+from openstack_query.query_server import QueryServer
+import openstack_query.manager_utils as manager_utils
+
+
+def search_all_servers(
+    cloud_account: str, query_details: QueryDetails
+) -> Union[str, List[Any]]:
+    return manager_utils.run_query(
+        cloud_account,
+        None,
+        query_details.properties_to_select,
+        query_details.group_by,
+        query_details.sort_by,
+    )
+
+
+def search_servers_older_than(
+    cloud_account: str,
+    query_details: QueryDetails,
+    days: int,
+    hours: int = 0,
+    minutes: int = 0,
+    seconds: int = 0,
```

Comment:
Do these defaults imply midnight just to double check?

Reply:
no these variables are used to define a time - days, hours, minutes, seconds since the current time

Reply:
it's poorly defined - I guess I should allow providing a timestamp instead

Reply:
I left this as is for now - best if we make an issue for this for a later PR since this one is big enough as is